maud_extras/lib.rs

```diff
@@ -160,3 +160,29 @@ impl<T: AsRef<str>, U: AsRef<str>> Render for MetaProperty<T, U> {
         }
     }
 }
+
+/// Generate a `<meta robots>` element.
+///
+/// # Example
+///
+/// ```rust
+/// # #![feature(plugin)]
+/// # #![plugin(maud_macros)]
+/// # extern crate maud;
+/// # extern crate maud_extras;
+/// # use maud_extras::*;
+/// # fn main() {
+/// let markup = html! { (MetaRobots("nofollow", "noindex")) };
+/// assert_eq!(markup.into_string(),
+///            r#"<meta name="robots" content="nofollow, noindex">"#);
+/// # }
+/// ```
+pub struct MetaRobots<T: AsRef<str>, U: AsRef<str>>(pub T, pub U);
+
+impl<T: AsRef<str>, U: AsRef<str>> Render for MetaRobots<T, U> {
+    fn render(&self) -> Markup {
+        html! {
+            meta name="robots" content=(format!("{}, {}", self.0.as_ref(), self.1.as_ref())) /
```

Comment:
Suggest writing this as

```rust
meta name="robots" content={
    (self.0.as_ref())
    ", "
    (self.1.as_ref())
} /
```

since this avoids using the `std::fmt` machinery.